Scenes/Resource.tscn

```diff
@@ -9,7 +9,7 @@
 
 [sub_resource type="ShaderMaterial" id=34]
 shader = ExtResource( 5 )
-shader_param/level = 1.01
+shader_param/level = 0.0435461
```

Comment:
why do you have changes in this file ? reset yor editor values

Reply:
I don't have any clue about why this changed.